docs/install.md

```diff
@@ -1,52 +1,113 @@
 # Install
 
-## **Under construction**
-
 ## Prerequisites
 
-### OCM based multi-cluster setup
-
-Ramen works as part of the [OCM hub] to orchestrate disaster-recovery, of
-application state, for applications deployed to [OCM managed clusters].
+### OCM managed multi-cluster setup
 
-As a result Ramen requires that multiple kubernetes clusters, that form the
-set of peer clusters, across which application deployment and state needs to
-be managed for disaster recovery are, managed clusters of an OCM hub.
+Ramen works as part of the [OCM hub](https://open-cluster-management.io/concepts/architecture/#hub-cluster)
+cluster to orchestrate the [placement](https://open-cluster-management.io/concepts/placement/)
+of [workloads](https://kubernetes.io/docs/concepts/workloads/) and their attachment
+to PersistentVolumes, on [OCM managed clusters](https://open-cluster-management.io/concepts/managedcluster/).
 
-The OCM hub orchestration is provided by Ramen using the
-[DRPlacementControl(DRPC) CRD](docs/drpc-crd.md).
+[Ramen hub](#ramen-hub) and [Ramen cluster](#ramen-cluster) components hence
+require an OCM managed multi-cluster setup for their operation.
 
-### Ceph-CSI as the volume provisioner
+### OCM Managed Cluster supporting VolumeReplication CRD
 
-Ramen also works as part of the [OCM managed clusters] to orchestrate,
+Ramen also works as part of the [OCM managed clusters](https://open-cluster-management.io/concepts/managedcluster/)
+to orchestrate,
```

Comment:
> The comma at the end of `orchestrate,` should perhaps be changed to a colon. What's listed below is core Ramen functionality; hence, the `also` in this sentence could imply additionally and doesn't seem appropriate given that it is core functionality.

ack!

> 
> I wrote the following for an internal IBM write up. I think the following can be added to the intro or in this section.

This is the install pre-req, so if we need to add/modify bullets it would be in the README.md for now I think.

> o Ramen integrates with RHACM to manage application-level placement control.
> o Ramen integrates with the storage layer like rook-ceph and other storage systems to replicate the PV’s storage volume to peer clusters.

We should not state we integrate with storage system layer, we integrate with VolumeReplication CSI extension, hence will steer away from this mention.

> o Ramen directly manages Persistent Volume (PV) related Kubernetes cluster data by backing it up to an object store suitably located on the peer clusters or peer cluster regions or in a region that is independent of all peers.

I want to stay away from mentioning PV at a very high level, and just state cluster data, as this may change as we progress (of course the comment may reflect to document what we do now, than later) and hence I mention PVs lower in the S3 section as a NOTE. The PV backup/restore is a current VolumeReplication requirement, and may change.

> o Ramen handles dynamic creation of PVs, recovery of an application to a peer cluster in the case of a disaster (aka failover) and relocation of an application to a peer cluster (aka failback), etc.
> 
> Given that people understand failover and failback better, I suggest using that at least in brackets. :-)

I think we need to veer away for this terminology, this is too DR centric and the intention is to steer away from that as the only use-case.